### PR TITLE
feat(W293): blank line contains whitespaces

### PR DESCRIPTION
**Pull request by carlosmiei**
- Added W293 from pycodestyle
- https://www.flake8rules.com/rules/W293.html

### CODE REVIEW

File: crates/ruff/src/settings/mod.rs [hunk 1]
```diff
@@ -480,7 +481,10 @@ mod tests {
 
         let actual = resolve_rules([RuleSelection {
             select: Some(vec![Pycodestyle::W.into()]),
```
Comment (carlosmiei) [inline crates/ruff/src/settings/mod.rs:483]: @charliermarsh Not 100% if I correctly adjusted these tests 😬

**Comment by carlosmiei** [comment]
Ok there's a PR recently with this rule already https://github.com/charliermarsh/ruff/pull/3122/files

**Comment by charliermarsh** [comment]
Oh yeah that's funny timing! Thank you for this.

**Comment by charliermarsh** [comment]
I could definitely use help with the remaining pycodestyle rules, FWIW.

**Comment by carlosmiei** [comment]
> I could definitely use help with the remaining pycodestyle rules, FWIW.

@charliermarsh sure, I will try to add the missing ones!

**Pull request closed without merging**